distributed/worker.py

```diff
@@ -488,6 +498,9 @@ def __init__(
         else:
             self.data = dict()
 
+        self.host = self.data.fast.d if hasattr(self.data, "fast") else self.data
+        self.disk = self.data.slow.d if hasattr(self.data, "slow") else dict()
```

Comment:
Ah, I see now the `else`.

I think that we should only have `memory` and `disk` attributes if they're using zict.  Otherwise I'm not too concerned.  Only expert users will turn off zict.  I'm still in favor of adding `self.data.memory` and `self.data.disk` only if we're in the branch above where we're creating the Buffer.  Otherwise I wouldn't bother.

My aversion to adding `.memory` and `.disk` to the top level `Worker` namespace is mostly that it's already crowded and that these are useful, but probably not that useful.  It's also less clear what they mean in that context.   It's also not something that we can guarantee if the user provides a different mapping type.

Reply:
> I think that we should only have `memory` and `disk` attributes if they're using zict. Otherwise I'm not too concerned. Only expert users will turn off zict. I'm still in favor of adding `self.data.memory` and `self.data.disk` only if we're in the branch above where we're creating the Buffer. Otherwise I wouldn't bother.

That's fine by me. But my previous comment is still valid, since `self.data` is a `zict.Buffer`, we can't simply add attribute `host` and `disk`, I think we would need to add a new object, something like `HostFile`. Or is there a simpler way of doing that you're thinking of?